addons/spreadsheet/static/src/chart/odoo_chart/odoo_chart.js

```diff
@@ -38,7 +38,14 @@ export class OdooChart extends AbstractChart {
     constructor(definition, sheetId, getters) {
         super(definition, sheetId, getters);
         this.type = definition.type;
-        this.metaData = definition.metaData;
+        this.metaData = {
+            ...definition.metaData,
+            mode: this.type.replace("odoo_", ""),
+            cumulated: definition.cumulative,
+            // if a chart is cumulated, the first data point should take into
+            // account past data, even if a domain on a specific period is applied
+            cumulatedStart: definition.cumulative,
```

Comment:
why is comulatedStart (a number) initialized as a bool ? Or am I missing something ?

Reply:
`cumulatedStart` is a boolean in the context of the entire chart definition (grep `cumulated_start="1"`) but the same property name is used for the actually amount (number) for the data points in the graph model

Reply:
Why do we need cumulatedStart And cumulated if we assume that every cumulated chart will have a cumulated start ? (or maybe I don't understand correctly the description in the task

Reply:
*"we"* here is "spreadsheet".
In the (web) graph view, it is supported to have `cumulated="1"` but `cumulated_start="0"` (currently never used). However, we only have one checkbox in the side panel. For the sake of simplicity of this bugfix, I only kept one option. If we want, we can add an other option in the side panel in master